O/OpenFHE_128/build_tarballs.jl

```diff
@@ -0,0 +1,18 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+
+# If you make changes to this file, be sure to update ../OpenFHE/build_tarballs.jl as well.
+include(joinpath(pwd(), "..", "OpenFHE", "common.jl"))
+
+name = "OpenFHE_128"
+version = v"1.2.3"
+
+# symlink patch file from OpenFHE
+mkpath(joinpath(pwd(), "bundled", "patches"))
+cp(joinpath(pwd(), "..", "OpenFHE", "bundled", "patches", "windows-fix-cmake-libdir.patch"),
+   joinpath(pwd(), "bundled", "patches", "windows-fix-cmake-libdir.patch"))
```

Comment:
Can't you just symlink the files directly in git? You might need to do this from a linux machine, but copying the files here seems weird to me...